src/common/layouts/Header/__tests__/Header.test.jsx

```diff
@@ -28,4 +30,34 @@ describe('Header', () => {
     const wrapper = shallow(<Header store={store} />).dive();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('does not show holdingpen if betauser is logged in', () => {
```

Comment:
`does not show authorized tool links if logged in user is not superuser nor cataloger`